### ISSUE TITLE
ignore sub-folders of origin.....

### ISSUE THREAD

**Issue by 4evermaat** [stefankueng/CryptSync#50]
I didn't see a `checkbox` or a `command line` that allows the user to ignore (skip) sub-folders from the origin folder.  But there are many use cases where you only want to sync folder contents, but ignore sub-folders.

There is an `enter files you dont want synced at all below`.

can you enter folder path here?  Or only file extensions?

There were no examples in the [website documentation](https://tools.stefankueng.com/CryptSync.html), so a bit hard to follow.

---

Still, there could be many subfolders, so a single switch to ignore all subfolders would be good idea.

**Comment by mkraemer0** [comment]
+1

And whitelists in addition to the blacklists would be nice.

**Comment by stefankueng** [comment]
you can ignore subfolders the same way as files with the `enter files you don't want synced at all below` field.
for example you can enter there:
`C:\mySynchedFolder\IgnoredFolder1|C:\mySynchedFolder\IgnoredFolder2`
or also just
`IgnoredFolder1|IgnoredFolder2`
